### DISCUSSION TITLE
Add "RC" (Remote Communities) as meeting type

### DISCUSSION THREAD

**Discussion by meetings03** [code4recovery/12-step-meeting-list#758] [Ideas]
Arizona Area uses a meeting type "RC" for Remote Communities and would like to see it added to the default list types.
Since we take feeds from several Intergroups around the state having RC as an option in the plugin would allow them to all be able to tag a meeting as RC which would then appear via their feed to our global Area meeting list.

**Comment by joshreisner** [comment]
it sounds like this type would be more in service of the service structure, rather than helping prospective alcoholics to find their first meeting. am i wrong about that?

as such i'm not really sure if it would make a good meeting type. the purpose is to help people find the meetings they want to attend. the more options we add here, the harder it is to find the really critical ones.

but technically you could achieve this result by adding the following code to each website (we recommend in the theme's functions.php file)

```
if (function_exists('tsml_custom_types')) {
    tsml_custom_types(array(
        'RC' => 'Remote Communities',
    ));
}
```

**Comment by meetings03** [comment]
Thanks Josh,
I've thought the same thing, service structure, but Area03 has always 
had an RC designation in there old meeting list and our RC Coordinator 
has been bugging me about how to add this in TSML so they can sort by RC 
for whatever purpose they need.
That said, at one point I had added the code to add RC as an option on 
our end but when I went to pull feeds from other Intergroup lists, RC 
disappeared as an option on our end?
Would our function.php get over written with TSML updates, thus removing 
the RC type code?
If so, that would explain the issue, probably logged in, saw and ran an 
update, then went to hookup feeds and assumed the feeds is what knocked 
the RC out on our end?
Any thought on that?

Thank You for being of service,
Greg S.
Area 03 Meeting List Coordinator

**Reply by kiyote33** [reply]
Hi Greg,
The function.php file where you could make code overrides is part of your theme package. Theme updates of course would remove your code overrides (without you knowing it if automatic updates are turned on). The use of child themes is recommended to avoid this situation. Simply put your code overrides in the child theme functions.php file, which will remain untouched when theme updates take place.  Personally, I've used several of the different child theme generator plugins from the WordPress directory with excellent results. Do a google search if you want to know more about child themes or related plugins.

Hope this helps...

PS... if you put your RC changes in the function.php file found in the \plugins\12-step-meeting-list\includes folder, then the changes would definitely be wiped out by a TSML update. As noted above, the suggested way to avoid that is to put them in the child theme function.php file.
